content/guides/typescript.md

```diff
@@ -136,3 +134,10 @@ declare module "*.svg" {
 ```
 
 Here we declare a new module for SVGs by specifying any import that ends in `.svg` and defining the module's `content` as `any`. We could be more explicit about it being a url by defining the type as string. The same concept applies to other assets including CSS, SCSS, JSON and more.
+
+## Performance Loader
+
+[`awesome-typescript-loader`](https://github.com/s-panferov/awesome-typescript-loader)
+
+Awesome TypeScript Loader has created a [wonderful explanation](https://github.com/s-panferov/awesome-typescript-loader#differences-between-ts-loader) of the difference between `awesome-typescript-loader` and `ts-loader`. The
+configuration for `awesome-typescript-loader` is more complex than `ts-loader`.
```

Comment:
This should just be joined with the previous line. At some point we may enforce a max line length (webpack-contrib/webpack-defaults#73) or something to keep it consistent but for now each paragraph should just be on one line for consistency. Then you can turn on word-wrapping in your editor if it's a pain to read.